_analyzers/token-filters/length.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Length
+parent: Token filters
+nav_order: 240
+---
+
+# Length token filter
+
+The `length` token filter is used to remove tokens that don't meet specified length criteria, such as minimum and maximum values, from the token stream.
```

Comment:
As written, this implies that there are sometimes other length criteria considered. Are there, or does this filter only consider the `min` and `max` values?

Reply:
Only min and max - reworded.